### DISCUSSION TITLE
How you contribute here?

### DISCUSSION THREAD

**Discussion by aregtech** [521xueweihan/HelloGitHub#1976] [聊一聊]
Hi,
sorry, I couldn't find any better place to as a questions :)

1. You have here many information. Do you have English version?
2. How can I contribute here? There is a newly release project, it is real-time communication engine to simplify distributed computing for embedded / IoT devices. Would you like to include here as well? https://github.com/aregtech/areg-sdk

**Comment by hx-w** [comment]
2. You can create an issue in this repo to introduce your project. Just follow the rules in [**this issue**](https://github.com/521xueweihan/HelloGitHub/issues/271) ( which can be translated into english by some tools ).

Hope to see your project in the next update :)

**Reply by aregtech** [reply]
Thank you replying. I'll try, hope it works and I'll manage :)

**Comment by ZeroXD123** [comment]
Messi o ronaldo?

**Comment by Yash-xoxo** [comment]
suiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
